### PR TITLE
Remove unused RBACs in GraphQL server

### PR DESCRIPTION
**Pull request by punithnayak**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ x] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
#4120

### CODE REVIEW

**Review by Saranya-jena — CHANGES_REQUESTED** [review]
go imports are not in order, please fix them by running this command: goimports -w -l ./

**Review by Saranya-jena — APPROVED** [review]
LGTM

File: chaoscenter/graphql/server/pkg/authorization/roles.go
Comment (namkyu1999) [inline chaoscenter/graphql/server/pkg/authorization/roles.go:1]: Great work! but I think we can remove more unused variables. Can you check out my [GitHub gist](https://gist.github.com/namkyu1999/d7eee6329ca93220b9f116ad3c01d3b5) and update this file?

**Review by namkyu1999 — CHANGES_REQUESTED** [review]

File: chaoscenter/graphql/server/pkg/authorization/roles.go
Comment (namkyu1999) [reply chaoscenter/graphql/server/pkg/authorization/roles.go:1]: > go imports are not in order, please fix them by running this command: goimports -w -l ./

backend-checks are failed. Can you try this one again? @punithnayak

Comment (punithnayak) [reply chaoscenter/graphql/server/pkg/authorization/roles.go:1]: Hey, can you check now?

Comment (namkyu1999) [reply chaoscenter/graphql/server/pkg/authorization/roles.go:1]: Resolving a merge conflict please @punithnayak

**Pull request closed without merging**